tutorials/fundamentals/typeof-array.md

```diff
@@ -0,0 +1,20 @@
+Arrays are a special case when using the `typeof` operator.
+To determine if an array is an array using `typeof`, you don't.
+Instead you should use `Array.isArray()`. The reason why is that
+if you were to use `typeof`, it would return `'object'`, not `'array'`.
+
+```javascript
+let array = [1,2,3,4];
+typeof array; // `object`
+```
+
+`Array.isArray()` takes one parameter and will return true only
+if the given value is an array. Anything else will return false.
```

Comment:
I'd also make a note about `Array.isArray()` with [subclassed arrays](http://thecodebarbarian.com/the-80-20-guide-to-javascript-arrays.html#subclassing-the-array-class)